diff --git a/ros_robot_controller/ros_robot_controller_node.py b/ros_robot_controller/ros_robot_controller_node.py
--- a/ros_robot_controller/ros_robot_controller_node.py
+++ b/ros_robot_controller/ros_robot_controller_node.py
@@ -180,6 +180,6 @@ class RosRobotController(Node):
                     data.offset = state
             if i.get_voltage:
-                state = self.board.bus_servo_read_voltage(i.id)
+                state = self.board.bus_servo_read_vin(i.id)
                 if state is not None:
                     data.voltage = state
             if i.get_temperature:
@@ -199,7 +199,7 @@ class RosRobotController(Node):
                 if state is not None:
                     data.max_temperature_limit = state
             if i.get_torque_state:
-                state = self.board.bus_servo_read_torque(i.id)
+                state = self.board.bus_servo_read_torque_state(i.id)
                 if state is not None:
                     data.enable_torque = state
             states.append(data)
